.github/workflows/weekly_tests.yaml

```diff
@@ -0,0 +1,22 @@
+name: Weekly Tests
```

Comment:
new workflow that only handles weekly CIs, which run:
- each sunday night
- on demand using GH's interface